homeassistant/components/climate/zwave.py

```diff
@@ -215,50 +219,29 @@ def set_temperature(self, **kwargs):
         else:
             return
 
-        for value in (self._node.get_values(
-                class_id=zwave.const.COMMAND_CLASS_THERMOSTAT_SETPOINT)
-                      .values()):
-            if value.index == self._index:
-                if self._zxt_120:
-                    # ZXT-120 responds only to whole int
-                    value.data = round(temperature, 0)
```

Comment:
Changes must have been made to OZW since I made the HVAC/climate component. Then it was not working without rounding. Now it responds perfectly to the set .5 intervals.